src/rgw/rgw_admin.cc

```diff
@@ -244,6 +244,8 @@ void usage()
   cout << "  datalog list               list data log\n";
   cout << "  datalog trim               trim data log\n";
   cout << "  datalog status             read data log status\n";
+  cout << "  datalog type               change datalog type\n";
+  cout << "  datalog prune              prune empty datalog generations\n";
```

Comment:
when/why would a user need to run this? can trimming not handle this automatically?

Reply:
I put it in for testing, basically. I wanted to make sure we were getting rid of the right generations.

Reply:
i'm fine with admin commands that aid testing, but not all of those should be documented for users. if it's unnecessary in normal operation, and more likely to cause problems than solve them, then i'd prefer not to document it

Reply:
Done.